articles/introduction-to-wireshark/index.md

```diff
@@ -0,0 +1,228 @@
+### Introduction to Network Analysis Using Wireshark
+As an IT professional, one of the most powerful networking tools you will find yourself using is ```Wireshark```, a tool mostly used in the analysis of network packets.  
+In this tutorial, we will be looking at an overview of ```Wireshark```, how it works, and several other uses. 
+
+### Requirements
+* Networking Basics
+* TCP/IP stack
+* Reading and interpreting packet headers
+* Routing and port forwarding
+* DHCP
+
+### What is Wireshark?
+
+Wireshark software's history goes back to 1998, as an open-source project started by volunteer network experts around the world.  
+The main aim was to build a standard network protocol analyzing tool.  
+
+Therefore, ```Wireshark``` is simply a network packet analyzer that is mainly used to capture the data on your network. This data is then presented in a form understandable by human beings.  
+
+In case you find yourself in a position where you need to try and troubleshoot every single network packet, ``` Wireshark``` is the right tool for you!
```

Comment:
```suggestion
In case you find yourself in a position where you need to try and troubleshoot network packets, _Wireshark_ is the right tool for you!
```